Libraries/Text/RCTTextAttributes.m

```diff
@@ -31,9 +31,13 @@ - (instancetype)init
     _textShadowRadius = NAN;
     _opacity = NAN;
     _textTransform = RCTTextTransformUndefined;
-#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
-    _foregroundColor = [NSColor labelColor];
-#endif // TODO(macOS ISS#2323203)
+    // [TODO(OSS Candidate ISS#2710739)
+    if (@available(iOS 13.0, *)) {
+      _foregroundColor = [RCTUIColor labelColor];
```

Comment:
Was this a purposeful logic change from _foregroundColor = labelColor on mac to _foregroundColor = blackColor?

Reply:
The asterisk indicates it will match any other platform that’s being targeted, besides the explicitly mentioned ones. So in the case of iOS it will match v13 and above, in the case of macOS it will match any version. i.e. the `else` is only for iOS < v13.